### PR TITLE
Add org.cubocore.CoreAction

### PR DESCRIPTION
**Pull request by s96Abrar**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by s96Abrar** [comment]
bot, build org.cubocore.CoreAction

File: org.cubocore.CoreAction.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+finish-args:
+  - --share=ipc
+  - --share=network # For weather and network plugin
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --system-talk-name=org.freedesktop.UPower # For battery plugin showing battery info
+  - --talk-name=org.kde.StatusNotifierWatcher # For showing in the system tray
+  - --talk-name=org.freedesktop.Notifications 
+  - --own-name=org.kde.StatusNotifierItem-2-1 # For showing in the system tray
```
Comment (barthalion) [inline org.cubocore.CoreAction.yml:15]:
```suggestion
  - --own-name=org.kde.* # For showing in the system tray
```

**Comment by s96Abrar** [comment]
bot, build org.cubocore.CoreAction

**Pull request closed without merging**